main.go

```diff
@@ -23,7 +23,7 @@ import (
 	"os"
 
 	// Import the snap plugin library
-	"github.com/intelsdi-x/snap-plugin-publisher-kafka/kafka"
+	"github.com/SmartX-Team/snap-plugin-publisher-kafka/kafka"
```

Comment:
Please, revert this change - there should be "github.com/intelsdi-x/snap-plugin-publisher-kafka/kafka" as import path